docs/_book.yaml

```diff
@@ -31,7 +31,8 @@ upper_tabs:
       skip_translation: true
       contents:
       - title: "All Symbols"
-        path: /reference/python/openfermion/all_symbols
-      - include: /reference/python/openfermion/_toc.yaml
+        path: /openfermion/api_docs/python/openfermion/all_symbols
```

Comment:
This is the wrong path

Reply:
@lamberta or @yashk2810 can you fix this?

Reply:
@MarkDaoust I think this PR can be reverted/closed?

Reply:
Yes.